client/boot/index.js

```diff
@@ -343,7 +343,7 @@ function reduxStoreReady( reduxStore ) {
 	}
 
 	if ( config.isEnabled( 'desktop' ) ) {
-		require( 'lib/desktop' ).init();
+		// require( 'lib/desktop' ).init();
```

Comment:
I can't reproduce this on wp-desktop master + HEAD. Can you share the full trace?